nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -58,20 +63,27 @@
  */
 public class S3DirectBatchHandler extends AbstractBatchHandler {
 
+    private static final Log log = LogFactory.getLog(S3BinaryManager.class);
+
     protected static final Pattern REGEX_MULTIPART_ETAG = Pattern.compile("-\\d+$");
 
+    protected static final Pattern REGEX_BUCKET_PATH_PLACE_HOLDER = Pattern.compile("\\{\\{bucketPath}}");
+
     // properties passed at initialization time from extension point
 
     public static final String ROLE_ARN_PROPERTY = "roleArn";
```

Comment:
The `ROLE_ARN_PROPERTY` constant is not used anymore, it can be removed (also from the `MANDATORY_PROPERTIES` list)